inc/softwareversion.class.php

```diff
@@ -329,8 +329,8 @@ static function showForSoftware(Software $soft) {
                echo "<td class='right'>".Dropdown::getDropdownName('glpi_operatingsystems',
                                                                    $data['operatingsystems_id']);
                echo "</td>";
-               echo "<td class='numeric'>$nb</td>";
-               echo "<td>".$data['comment']."</td></tr>\n";
+               echo "<td class='numeric'>$nb</td>";               
```

Comment:
warning, you added some trailing spaces after the end of this line (tests fail because that)

Reply:
Apologies. First time I've done this on github.